docs/Plan/02. kpi_measurement_design.md

```
@@ -153,157 +268,122 @@ SearchWeb PRD에서 정의된 5개 KPI 카테고리의 **목표 수치를 실제
+
+    eventService.track(memberId, "bookmark_saved", Map.of(
         "bookmark_id", result.getId(),
         "has_ai_tag", request.hasAiTag(),
         "ai_tag_count", request.getAiTagCount(),
         "url_domain", extractDomain(request.getUrl())
     ));
-    
+
     return result;
 }
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)